packages/generators/templates/component/{{name}}.{{mdx}}

```diff
@@ -47,10 +38,69 @@ When contributing to, or consuming the {{ name }} component, consider the follow
   See `Button.mdx` for a good example of this.
 -->
 
-## Loud
+## Usage Guidelines
 
-Use `loud` to indicate that a component is yelling.
+The <{{name}}> will... {Add a brief description of the component}
 
-<Playground>
-  <{{ name }} text="Fred" loud={true} />
-</Playground>
+<!--
+  What is the design purpose of this component? How do its responsibilities
+  relate to other components? What should this component not be used for? What
+  are some related components in Atlantis? Try to describe this component not
+  in terms of what it looks like or what elements make it up, but what function
+  it performs or how it enables a user to perform tasks (e.g. “Buttons allow users
+  to initiate, complete, and reverse actions.” vs “Buttons are rectangular elements
+  that have a green background and white text”).
+-->
+
+## Content Guidelines
+
+<!--
+  What type of content does this component present (documents, text, images, other components)?
+  What is the recommended, or expected, amount of content? What is the components’ behaviour when
+  the provided content diverges from what is expected (image aspect ratio, amount of text, video
+  vs audio, filesize)? Are there any important references to the [Product Vocabulary](https://atlantis.getjobber.com/guides/product-vocabulary)
+  that should be made regarding this component? What happens when content is unavailable or unreliable
+  (poor connection, empty states, flaky GPS data?)
+-->
+
+## Accessibility
+
+<!--
+  Describe the accessibility concerns for the component. How does it handle touch vs cursor vs
+  keyboard events? Should it capture input? What should a screen reader see when it's focuse?
+-->
+
+## Responsiveness
+
+<!--
+  How should the component behave on an iPhone 8? An iPad? A 1920x1080 monitor? How does the
+  component appear when the component itself is less than 375px wide? Less than 640px wide?
+  Less than 1200px wide? Greater than 1200px? Does it change when the device DPI is higher
+  or lower?
+-->
+
+## Mockup
+
+<iframe
+  style="border: 1px solid rgba(0, 0, 0, 0.1);"
+  width="800"
+  height="450"
+  src="https://www.figma.com/embed?embed_host=share&url=https%3A%2F%2Fwww.figma.com%2Ffile%2F3BXCXXD9glMtj8RAHjXiQC%2FDesign-System-Contribution-TEMPLATE%3Fnode-id%3D332%253A5526"
+  allowfullscreen>
+</iframe>
```

Comment:
I strongly beleive that we should make this into a component that looks something like

```
<Figma src="" />
```

maybe not with this PR, but a backlog item for this would be nice.

Reply:
https://jobber.atlassian.net/browse/JOB-30156